### ISSUE TITLE
Reduce the transaction timeout from 200ms to something shorter

### ISSUE THREAD

**Issue by ifreund** [riverwm/river#998]
The transaction timeout is how long river will wait for clients to respond to configure events before giving up on frame perfection and rendering an imperfect frame. I think 200ms feels too long, for me at least it is beyond the point where river's efforts at frame perfection end up just making everything feel sluggish.

I think a value between 50ms and 100ms would make more sense, but it's important to test on a range of hardware to make sure we don't make things worse for people on slower hardware. 

I'd really rather not make this configurable, I think there is a reasonable value that should work fine for everyone.

If anyone wants to play with different timeouts, the constant to tweak is here:
https://github.com/riverwm/river/blob/f2b6893b00a089810c94a86fa2c8df1b823fcaf4/river/Root.zig#L585

**Comment by novakne** [comment]
For me at 50ms, I haven't see any problems, below that, at 25, I'm starting to get a lot of `error(transaction): timeout occurred, some imperfect frames may be shown`.
I'll keep testing at 50 ms to be sure

A bit of `lscpu` output, also 8gb of RAM
```
Architecture:            x86_64
  CPU op-mode(s):        32-bit, 64-bit
  Address sizes:         39 bits physical, 48 bits virtual
  Byte Order:            Little Endian
CPU(s):                  4
  On-line CPU(s) list:   0-3
Vendor ID:               GenuineIntel
  Model name:            Intel(R) Core(TM) i7-6500U CPU @ 2.50GHz
    CPU family:          6
    Model:               78
    Thread(s) per core:  2
```

**Comment by motato1** [comment]
> I think a value between 50ms and 100ms would make more sense, but it's important to test on a range of hardware to make sure we don't make things worse for people on slower hardware.

@ifreund Is there a specific way to test this?

I am happy to help, but I couldn't notice any difference when using different values (tested values from 10 to 2000). I didn't do much more than open a few programs and look for anything that might be "off" (delays, flickering, etc.)

**Comment by ifreund** [comment]
Implemented in 4a65af667e59a703000d859e77098a4893b77fa6

> I am happy to help, but I couldn't notice any difference when using different values (tested values from 10 to 2000). I didn't do much more than open a few programs and look for anything that might be "off" (delays, flickering, etc.)

This timeout should only be hit when clients take longer than X ms to respond to one of river's configure events and commit a new buffer. The symptom of the timeout being hit is that there will be a temporary "hole" in the layout where the client that missed the deadline should be until it finally commits a new buffer. This kind of frame imperfection is obviously not desirable, but freezing the rendered layout for 200ms to wait for a client to respond is long enough to feel quite sluggish. The new timeout of 50ms feels like a better tradeoff.
